<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAdvanceSetting</class>
 <widget class="QDialog" name="DlgAdvanceSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>705</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>705</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>个性化设置</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>1</y>
     <width>32</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_support">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>667</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>24</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_personal">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>671</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>117</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>117</width>
     <height>16</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_menu">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>667</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>70</horstretch>
     <verstretch>24</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_progress_bar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>667</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_launch_bg">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>667</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_pe_bg">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>667</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>24</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_restore_default">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>667</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>667</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>40</horstretch>
     <verstretch>24</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>667</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>40</horstretch>
     <verstretch>24</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
